packages/component-base/src/dom-utils.js

```diff
@@ -53,10 +55,15 @@ function deserializeAttributeValue(value) {
 }
 
 /**
+ * Takes a set of string values and returns a string with values separated by space
+ *
  * @param {Set<string>} values
  * @return {string}
  */
-function serializeAttributeValue(values) {
+export function serializeAttributeValue(values) {
+  if (!values) {
+    return '';
+  }
```

Comment:
This could be actually simplified using a ternary operator.

Reply:
Refactored.